src/main/java/uk/gov/hmcts/cmc/claimstore/documents/ClaimIssueReceiptService.java

```diff
@@ -35,4 +36,9 @@ public ClaimIssueReceiptService(
         );
     }
 
+    @Override
+    public String filename(Claim claim) {
+        return buildClaimIssueReceiptFileBaseName(claim.getReferenceNumber());
```

Comment:
are we missing file extension in file name here?

Reply:
no, in these ones we append the extension when we create the PDF (doing `new PDF(bla)` appends the extension automatically; I find these different approaches a bit confusing